package.json

```diff
@@ -112,6 +112,7 @@
     "@typescript-eslint/eslint-plugin": "1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
     "cypress": "^3.4.1",
+    "jest-date-mock": "^1.0.7",
```

Comment:
Should we add this to `setupFiles` per the [README](https://github.com/hustcc/jest-date-mock#setup)?

Reply:
I tried but that requires ejecting from create-react-app and it looked unnecessary for what we're doing.